frappe/custom/doctype/custom_field/custom_field.py

```diff
@@ -135,7 +135,7 @@ def create_custom_field(doctype, df, ignore_validate=False):
 		})
 		custom_field.update(df)
 		custom_field.flags.ignore_validate = ignore_validate
-		custom_field.insert()
+		custom_field.insert(db_auto_commit = frappe.flags.in_test or frappe.flags.in_install or frappe.flags.in_setup_wizard)
```

Comment:
Same comment as I've added in `property_setter.py`

Reply:
This is now removed